domino/domino.py

```diff
@@ -96,9 +96,11 @@ def runs_start(self, command, isDirect=False, commitId=None, title=None,
             "tier": tier,
             "publishApiEndpoint": publishApiEndpoint
         }
-
-        response = self.request_manager.post(url, json=request)
-        return response.json()
+        try:
+            response = self.request_manager.post(url, json=request)
+            return response.json()
+        except ReloginRequiredException:
+            self._logger.info(f" You need to log in to the Domino UI to start the run. Please do it at {self._routes.host}")
```

Comment:
I would slightly modify the url to be something like `{self._routes.host}/relogin?redirectPath=/